src/widgets/PaymentsList/PaymentsListTable/PaymentsListRow.tsx

```
@@ -0,0 +1,40 @@
+import { SubscriptionPaymentsModel } from '@/types'
+import { Avatar, TableCell, TableRow, Typography } from '@funnyteam/ui-kit'
+
+import s from './PaymentsListTable.module.scss'
+
+type PropsType = {
+  payment: SubscriptionPaymentsModel
+}
+
+export const PaymentsListRow = ({ payment }: PropsType) => {
+  const paymentAvatar =
+    (payment && payment.avatars && payment.avatars.length !== 0 && payment.avatars[0].url) || ''
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)